test/Riok.Mapperly.Tests/Mapping/NamedMappingTest.cs

```diff
@@ -135,4 +135,93 @@ public Task MapPropertyFromSourceWithUseUsesNamedMapping()
 
         return TestHelper.VerifyGenerator(source);
     }
+
+    [Fact]
+    public Task MapPropertyWithUseUsesNamedMappingWithExternalMapping()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypesInBlockScopedNamespace(
+            """
+            [MapProperty(nameof(A.StringValue), nameof(B.StringValue), Use = "OtherNamespace.OtherMapper.CustomModifyString")]
+            public partial B Map(A source);
+            """,
+            TestSourceBuilderOptions.Default,
+            "class A { public string StringValue { get; set; } }",
+            "class B { public string StringValue { get; set; } }"
+        );
```

Comment:
I was wondering if NamedMapping and external mapping can be combined. It just needed a slight enchantment, so I did it.